translations/messages.fr.yml

```diff
@@ -16,6 +16,8 @@ approved: Approuvé
 #
 Invalid credentials.: L'adresse e-mail et le mot de passe que vous avez saisis ne correspondent pas.
 Max attempts reached.: Vous avez effectué 5 tentatives de connexion erronées. Veuillez attendre 1 minute avant de réessayer.
+adherent.error.must_be_validated: Pour vous connecter vous devez confirmer votre adhésion. Si vous n'avez pas reçu le mail de validation, vous pouvez <a href="%url%">cliquer ici</a> pour le recevoir à nouveau.
+Account is disabled.: Votre compte a été désactivé.
```

Comment:
why do you override the existing translation of this message?
`Le compte est désactivé.` cf `security.fr.xlf`

Reply:
oups :D